packages/espressocash_app/lib/features/kyc_sharing/screens/document_input_screen.dart

```diff
@@ -0,0 +1,554 @@
+// ignore_for_file: avoid-recursive-calls
```

Comment:
Don't turn off linters like this on file level, this only makes sense for things like invalid annotation target in files with DTOs.

Otherwise, if you need it, put inline ignore with the description why it's turned off.

Reply:
updated code. refactored to not use recursive calls